build.gradle

```diff
@@ -16,6 +16,8 @@ subprojects { subproject ->
     sourceCompatibility = '1.8'
     targetCompatibility = '1.8'
 
+    compileJava.options.encoding = 'UTF-8'
```

Comment:
this still needs to go

Reply:
Done. I hope now i have every thing alright.